main.go

```diff
@@ -3,44 +3,68 @@ package main
 import (
 	"context"
 	"errors"
+	"flag"
```

Comment:
Why is this file being moved? Is there convention for it to live in the root?

Reply:
The main.go is moved to the root directory to align with the 1.1 project layout: https://v1-1-x.sdk.operatorframework.io/docs/building-operators/golang/migration/#what-was-changed